### PR TITLE
foxit-pdf-reader: Add version 11.0.1.49938

### PR DESCRIPTION
**Pull request by gitolicious**
Deprecation note added in #6584

### CODE REVIEW

File: bucket/foxit-pdf-reader.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "bin": "FoxitPDFReader.exe",
+    "shortcuts": [
+        [
+            "FoxitPDFReader.exe",
+            "Foxit PDF Reader"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.foxit.com/pdf-reader/version-history.html",
+        "re": "Version ([\\d.]+)"
```
Comment (Ash258) [inline bucket/foxit-pdf-reader.json:23]:
```suggestion
        "regex": ">Version ([\\d.]+)</"
```

File: bucket/foxit-pdf-reader.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "autoupdate": {
+        "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/$majorVersion.$minorVersion.$patchVersion/FoxitPDFReader$majorVersion$minorVersion$patchVersion_enu_Setup_Prom.exe"
```
Comment (Ash258) [inline bucket/foxit-pdf-reader.json:26]:
```suggestion
        "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/$matchHead/FoxitPDFReader$majorVersion$minorVersion$patchVersion_enu_Setup_Prom.exe"
```

File: bucket/foxit-pdf-reader.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "homepage": "https://www.foxit.com/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.foxitsoftware.com/pdf-reader/eula.html"
+    },
+    "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/11.0.1/FoxitPDFReader1101_enu_Setup_Prom.exe",
+    "hash": "5409b9966d344ea280d076c1c5feeec7e89e75249fb8f0f5b8ba31e894939b58",
+    "installer": {
+        "script": "Get-Item \"$dir\\Foxit*prom.exe\" | Expand-InnoArchive -Destination \"$dir\" -Removal"
+    },
```
Comment (Ash258) [inline bucket/foxit-pdf-reader.json:13]:
```suggestion
    "innosetup": true,
```

File: bucket/foxit-pdf-reader.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "11.0.1.49938",
+    "description": "View, annotate, form fill, and sign PDF across desktop, mobile, and web – no matter if you’re at the office, home, or on the go.",
+    "homepage": "https://www.foxit.com/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.foxitsoftware.com/pdf-reader/eula.html"
```
Comment (Ash258) [inline bucket/foxit-pdf-reader.json:7]:
```suggestion
        "url": "https://www.foxit.com/pdf-editor/eula.html"
```

File: bucket/foxit-pdf-reader.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "11.0.1.49938",
+    "description": "View, annotate, form fill, and sign PDF across desktop, mobile, and web – no matter if you’re at the office, home, or on the go.",
+    "homepage": "https://www.foxit.com/",
```
Comment (Ash258) [inline bucket/foxit-pdf-reader.json:4]:
```suggestion
    "description": "Fast and feature rich PDF reader that offers a delightful reading experience in a small footprint.",
    "homepage": "https://www.foxit.com/pdf-reader/",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/foxit-pdf-reader.json (hunk 5 above)
Comment (gitolicious) [reply bucket/foxit-pdf-reader.json:4]: The times when Foxit had a small footprint are over...

File: bucket/foxit-pdf-reader.json (hunk 1 above)
Comment (gitolicious) [reply bucket/foxit-pdf-reader.json]: Done

File: bucket/foxit-pdf-reader.json (hunk 2 above)
Comment (gitolicious) [reply bucket/foxit-pdf-reader.json]: Done

File: bucket/foxit-pdf-reader.json (hunk 4 above)
Comment (gitolicious) [reply bucket/foxit-pdf-reader.json]: It's weird that the Reader is forwarding to the Editor's EULA, but it seems to be true... Done

File: bucket/foxit-pdf-reader.json (hunk 5 above)
Comment (gitolicious) [reply bucket/foxit-pdf-reader.json]: Homepage is changed. I would suggest to keep the new description.

File: bucket/foxit-pdf-reader.json (hunk 3 above)
Comment (gitolicious) [reply bucket/foxit-pdf-reader.json]: Done

**Review by Ash258 — APPROVED** [review]

**Comment by gitolicious** [comment]
No problem from Germany...

image

How about [cdn02](https://cdn02.foxitsoftware.com/product/reader/desktop/win/11.0.1/FoxitPDFReader1101_enu_Setup_Prom.exe) for you?

**Comment by gitolicious** [comment]
Is download speed still an issue for you?

**Comment by issaclin32** [comment]
> No problem from Germany...

No problem from Taiwan either (3MB/s)

**Comment by rashil2000** [comment]
image

Quite slow for me (120Mbps fibre)

**Comment by issaclin32** [comment]
> Quite slow for me (120Mbps fibre)

Does [cdn07](https://cdn07.foxitsoftware.com/product/reader/desktop/win/11.0.1/FoxitPDFReader1101_enu_Setup_Prom.exe) or [cdn09](https://cdn09.foxitsoftware.com/product/reader/desktop/win/11.0.1/FoxitPDFReader1101_enu_Setup_Prom.exe) work better?
Both are fine (>2MB/s) in Taiwan.

**Comment by niheaven** [comment]
What about [cdn06](https://cdn06.foxitsoftware.com/product/reader/desktop/win/11.0.1/FoxitPDFReader1101_enu_Setup_Prom.exe)? It redirects to cdn06.foxitsoftware.com.cdn.cloudflare.net and may has better speed.

**Comment by rashil2000** [comment]
cdn07 is unreachable
cdn09 is slightly better (230Kbps)

cdn06 is 6Mbps, works best

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**